### PR TITLE
fix(workflow): poetry install failed

### LINKED ISSUE #130: Pylint Checks pipeline failed to install dbus-python
### Description

When CD pipeline run it fails on `dbus-python` installation.

```
Backend subprocess exited when trying to invoke build_wheel
  
  Traceback (most recent call last):
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
      main()
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/pyproject_hooks/_in_process/_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/pyproject_hooks/_in_process/_in_process.py", line 251, in build_wheel
      return _build_backend().build_wheel(wheel_directory, config_settings,
    File "/tmp/tmpe0mkwz6_/.venv/lib/python3.9/site-packages/mesonpy/__init__.py", line 985, in wrapper
      return func(*args, **kwargs)
    File "/tmp/tmpe0mkwz6_/.venv/lib/python3.9/site-packages/mesonpy/__init__.py", line 1038, in build_wheel
      with _project(config_settings) as project:
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/contextlib.py", line 119, in __enter__
      return next(self.gen)
    File "/tmp/tmpe0mkwz6_/.venv/lib/python3.9/site-packages/mesonpy/__init__.py", line 912, in _project
      yield Project(source_dir, build_dir, meson_args, editable_verbose)
    File "/tmp/tmpe0mkwz6_/.venv/lib/python3.9/site-packages/mesonpy/__init__.py", line 635, in __init__
      self._meson = _get_meson_command(pyproject_config.get('meson'))
    File "/tmp/tmpe0mkwz6_/.venv/lib/python3.9/site-packages/mesonpy/__init__.py", line 947, in _get_meson_command
      meson_version = subprocess.run(cmd + ['--version'], check=False, text=True, capture_output=True).stdout
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/subprocess.py", line 505, in run
      with Popen(*popenargs, **kwargs) as process:
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/subprocess.py", line 951, in __init__
      self._execute_child(args, executable, preexec_fn, close_fds,
    File "/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/subprocess.py", line 1837, in _execute_child
      raise child_exception_type(errno_num, err_msg, err_filename)
  FileNotFoundError: [Errno 2] No such file or directory: 'meson'
  

  at /opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/poetry/installation/chef.py:166 in _prepare
      162│ 
      163│                 error = ChefBuildError("\n\n".join(message_parts))
      164│ 
      165│             if error is not None:
    → 166│                 raise error from None
      167│ 
      168│             return path
      169│ 
      170│     def _prepare_sdist(self, archive: Path, destination: Path | None = None) -> Path:

Note: This error originates from the build backend, and is likely not a problem with poetry but with dbus-python (1.3.2) not supporting PEP 517 builds. You can verify this by running 'pip wheel --no-cache-dir --use-pep517 "dbus-python (==1.3.2)"'.
```

I think there is a problem with poetry >= v1.7; I try with poetry v1.6.1 it's working well for me.

### PR DESCRIPTION
**Pull request by nlevee**
Fix #130 

You can check : 
https://github.com/nlevee/gnome-next-meeting-applet/actions/runs/6940927222/job/18880752851

### CODE REVIEW

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -9,10 +9,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Install poetry pylint
+    - name: Install system dependencies
+      run: sudo apt install gobject-introspection pkg-config libcairo-dev libgirepository1.0-dev libdbus-1-dev
+    - name: Install poetry
+      run: pipx install poetry==v1.6.1
```
Comment (chmouel) [inline .github/workflows/pythonapp.yml:15]: That pins poetry to 1.6.x right ? I don't mind this for CI but I guess we are still broken for users who has that version installed

**Comment by chmouel** [comment]
Thanks for the fix I still need to figure out about poetry 1.7.x breakage as well to make a release on my desktop (on arch which has this version as well)

File: .github/workflows/pythonapp.yml (hunk 1 above)
Comment (nlevee) [reply .github/workflows/pythonapp.yml:15]: Yes it fix poetry version. 

I don't find why it's not working, but I'm not an expert with python an poetry.

I'm gonna ask

Comment (chmouel) [reply .github/workflows/pythonapp.yml:15]: ah if you find out that would be great!

**Comment by nlevee** [comment]
You can make a virtual env to install poetry at this version manually. But we have to find why it's not working 😕

**Comment by chmouel** [comment]
Yeah, it’s a bit awkward tho to have to run a virtualenv to manage a tool that manages virtualenv 🙃😅😅

**Pull request merged**